packages/core/src/modules/proofs/ProofsModule.ts

```diff
@@ -137,8 +137,6 @@ export class ProofsModule {
     connection.assertReady()
 
     const proofRequestFromProposalOptions: ProofRequestFromProposalOptions = {
-      name: options.config?.name ? options.config.name : 'proof-request',
-      version: options.config?.version ?? '1.0',
       nonce: await service.generateProofRequestNonce(),
```

Comment:
I don't have a "Resolve Conversation" button for this PR for some reason (perhaps because I am working on Prasad's fork) so I will just add a comment when each issue is done. Something along the lines of...Done.

Done.